build.gradle

```diff
@@ -13,6 +13,9 @@ version = '4.0.0-SNAPSHOT'
 allprojects {
     repositories {
         mavenCentral()
+        maven { url "https://repo.spring.io/milestone" }
```

Comment:
Are the repo.spring.io/milestone and snapshot needed? Worked locally for me without them, but the shibboleth repo was necessary. 

Lets look at SAML dependencies more later in a future PR.

Reply:
Tested the implementation without these dependencies and it worked, so I removed them.